src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -720,7 +720,7 @@ private void RunConnectRequestHandlerTask(ConnectParams connectParams)
                     }
 
                     // open connection based on request details
-                    result = await Instance.Connect(connectParams);
```

Comment:
>Instance [](start = 35, length = 8)

why are we just removing Instance in a couple cases and not all the cases.  I'm assuming the motivation from removing instance is that someday we'll have MEF create the singleton and then Instance would be the wrong actual instance?